mongodbatlas/resource_mongodbatlas_advanced_cluster_test.go

```diff
@@ -581,7 +581,7 @@ resource "mongodbatlas_advanced_cluster" "test" {
   project_id   = %[1]q
   name         = %[2]q
   cluster_type = "REPLICASET"
-  retain_backups_enabled = "true"
+  retain_backups_enabled = "false"
```

Comment:
sorry, I missed this question: The `retain_backups_enabled` has the following properties

- it is set as part of the cluster creation
- it cannot be updated
- it is not returned by the cluster API, this is a param that TF provides when deleting a cluster to keep the backups of the cluster once deleted

If the test imports the same cluster that was created with `retain_backups_enabled = true` in the previous step of the test, it will fail since there is no API to set this param to true on Atlas.